docs/vertx-runtime/master.adoc

```diff
@@ -69,12 +72,18 @@ include::topics/assembly_building-and-deploying-the-booster-to-openshiftlocal.ad
 :context: {parameter-mission}-ocp
 include::topics/assembly_building-and-deploying-the-booster-to-openshiftcontainerplatform.adoc[leveloffset=+3]
 
-
 include::topics/configmap-mission-basic-interaction-vertx.adoc[leveloffset=+3]
 
+include::topics/proc_running-integration-tests.adoc[leveloffset=+3]
+
 include::topics/configmap-mission-resources.adoc[leveloffset=+3]
 
 
+:parameter-mission name!:
```

Comment:
I am 99% sure this is wrong, at least because of the space.

Reply:
true, my bad.. fixes inbound